src/preprocess/base.py

```diff
@@ -161,14 +173,17 @@ def chop_roi(ds: xr.Dataset,
 
     def merge_files(self, subset_str: Optional[str] = 'kenya',
                     resample_time: Optional[str] = 'M',
-                    upsampling: bool = False) -> None:
+                    upsampling: bool = False,
+                    variable: Optional[str] = None) -> None:
 
         ds = xr.open_mfdataset(self.get_filepaths('interim'))
 
         if resample_time is not None:
             ds = self.resample_time(ds, resample_time, upsampling)
 
-        out = self.out_dir / f'{self.dataset}\
-        {"_" + subset_str if subset_str is not None else ""}.nc'.replace(' ', '')
+        out = self.out_dir / f'{self.dataset}' \
```

Comment:
Same question as above - why is this necessary? Where is it used?

Reply:
ah this got muddled with the ERA5land stuff sorry :( 

It was supposed to deal with the fact that there are so many different era5 land variables that itmakse sense to preprocess them separately

Reply:
should we move this over to the era5 land PR?

Reply:
I think what makes more sense is to allow the merge files to be overridden completely, rather than having these new options that don't apply to all preprocessors